exampleSite/content/content/emoji-support.md

```diff
@@ -41,7 +41,7 @@ To enable emoji globally, set `enableEmoji` to `true` in your site's [configurat
 <p><span class="nowrap"><span class="emojify">🙈</span> <code>:see_no_evil:</code></span>  <span class="nowrap"><span class="emojify">🙉</span> <code>:hear_no_evil:</code></span>  <span class="nowrap"><span class="emojify">🙊</span> <code>:speak_no_evil:</code></span></p>
 <br>
 
-The [Emoji cheat sheet](http://www.emoji-cheat-sheet.com/) is a useful reference for emoji shorthand codes.
+The [Emoji cheat sheet](https://www.webfx.com/tools/emoji-cheat-sheet/) is a useful reference for emoji shorthand codes.
```

Comment:
It looks like the domain redirects to that page anyway. I'd like to keep the original Hugo demo content for now.

Reply:
> I'd like to keep the original Hugo demo content for now.

I see. Then I would propose to switch to https here: https://www.emoji-cheat-sheat.com

Reply:
Sounds good.